files/default/agent.py

```diff
@@ -169,15 +162,12 @@ def send(self):
             system_status_to_delete = []
             try:
                 logger.debug("Creating heartbeat reply...")
-                disk_info = DiskInfo()
-                memory_info = MemoryInfo()
-                load_info = LoadInfo()
                 services_list = self.construct_services_status()
-                now = long(time.mktime(datetime.now().timetuple()))
+                now = int(time.mktime(datetime.now().timetuple()))
                 headers = {'content-type': 'application/json'}
                 headers['Authorization'] = "Bearer " + master_token
                 payload = {}
-                payload["num-gpus"] = devices.get_num_gpus()
+                payload["num-gpus"] = 0
```

Comment:
I'm not familiar with kagent code, but won't hard-coding num-gpus to 0 have any implications somewhere else?

Reply:
Nah, we don't really use this code anymore. What you see in the UI when you run with GPUs come from K8s, not Kagent.